ui/src/components/DiffTextArea/index.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { string, func, number } from 'prop-types';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import TextField from '@material-ui/core/TextField';
+import { withStyles } from '@material-ui/core/styles';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import { ReactGhLikeDiff } from 'react-gh-like-diff';
+import './index.css';
```

Comment:
To import the css, we can do `import 'react-gh-like-diff/lib/diff2html.css';` and remove `index.css`. To remove the file header section of the diff, we adjust the global css in `Main.jsx`:

```jsx
'@global': {
  // ...
  '.d2h-file-header': {
     display: 'none',
  },
  // ...
},
```

Reply:
I used `index.css` not only because I want to remove header section but also want to handle the `dark theme`. I adjusted some colors to fit it. Maybe we can add a `flag` to decide which `css` should import? `React.lazy()` might to the trick.

Reply:
Or maybe I can add style in `theme.js` and use `isDarkTheme` to decide which color I want to use.

Reply:
Never mind, I think I can use `theme.palette.type` to determine it is dark or light mode now and then change the color. I'll give it a try and update PR later, thanks for the review!